### ISSUE TITLE
Make release for branch - round clock testing

### ISSUE THREAD

**Issue by kolmanv** [bbfsdev/bbfs#114]
1) Update gems.
2) Set proper label/tag in git.

**Comment by kolmanv** [comment]
Blocked by issue #111 (gen dependencies check).

**Comment by genadyp** [comment]
Minor remark:
In all gemspecs need to replace a homepage:
s.homepage    = 'http://github.com/kolmanv/bbfs'
to 
s.homepage    = 'http://github.com/bbfsdev/bbfs'
Also think about replacing contact mail and owner.

**Comment by yarondbb** [comment]
New branch 
      from: round clock testing 
       to:   round_clock_testing_release_1_0_0_win - ignore the win (it is valid now for both windows and unix)
1. unpdate all s.homepage = 'http://github.com/bbfsdev/bbfs'
1. updated all modules revisions to 1.0.0
2. Build and installed all gems
3. Executed - content server using gems only. Execution works OK. @kolmanv , maybe we want to test round clock with it for a day ot two?
4. Prepared local intallation scripts:
     Windows:
         bbfs/gem_build_win.bat - Will install bundle, all infrastracture gems and content_server gems.
         bbfs/gem_rebuild_win.bat - same as gem_build_win.bat, but will also uninstall ALL gems first.
         run: 
             type: bbfs\gem_build_win.bat
             type: bbfs\gem_rebuild_win.bat
   
     Linux:
         bbfs/gem_build_lnx - Will install bundle, all infrastracture gems and content_server gems.
         bbfs/gem_rebuild_lnx - same as gem_build_win.bat, but will also uninstall ALL gems first.
         run: 
             type: /bin/sh gem_build_lnx
             type: /bin/sh gem_rebuild_lnx
5. pushed the branch to github

## Issues I have facing (reassign)
1. I am unable to push gems (tried from Windows), I get:
   C:\Users\ydror1\Ruby\Git\bbfs>gem push log
   Enter your RubyGems.org credentials.
   Don't have an account yet? Create one at http://rubygems.org/sign_up
   Email:   bbfsdev@gmail.com
   Password:
   Pushing gem to https://rubygems.org...
   HTTP Basic: Access denied.
2. I cannot push from Linux to remote tracking branch.. I get some error (This is the first time I try to do this)

**Comment by yarondbb** [comment]
1. @kolmanv , pls push the gems. and Start to round clock test the gems?
2. The issue is not urgent. I have opend new issue #123 for this.
3. Also, I have found out that, when install content_server gem, other dependent gems are not installed with it.
   The root cuase is the run time dependencies used in it..
   Currently I wrote a script to install all the gems and content sergver gem as the last one.

**Comment by kolmanv** [comment]
The third point is not clear. May be a problem. Lets discuss this next week.

**Comment by yarondbb** [comment]
I have added dependencies in order for 'gem install content_server-1.0.0.gem' will install all necessary gems to execute content server. Changes pushed to branch.
No need for the bundler.

**Comment by kolmanv** [comment]
What branch you pushed the changes? master? round_clock_testing?
Should be both.
Yaron have you marked this branch as release 1.0.0 in github?

**Comment by kolmanv** [comment]
@yarondbb (Yaron), what is the status of your task, are you blocked? Can you finish it? What are your dependencies?

**Comment by AlexeyNemytov** [comment]
@yarondbb 
@kolmanv 
Firiends I tried to install version 1.0.0 on f1
This is the result:

-bash-4.1$ gem install content_server
Fetching: content_server-0.0.10.gem (100%)
Successfully installed content_server-0.0.10
1 gem installed
Installing ri documentation for content_server-0.0.10...
Installing RDoc documentation for content_server-0.0.10...

**Comment by yarondbb** [comment]
Try now :)
I have succesfully pushed the gems from kolman user
Task is done from my side
Kolman pls verify:
Uninstall all gems - type (linux):  
    for x in `gem list --no-versions`; do gem uninstall $x -a -x -I; done
 Install content server - Type:
    gem install content_server

**Comment by kolmanv** [comment]
Alexey, please validate and close the bug.

**Comment by kolmanv** [comment]
@AlexeyNemytov

**Comment by kolmanv** [comment]
@AlexeyNemytov Is this issue closed?

**Comment by AlexeyNemytov** [comment]
issue validated and closed
